### DISCUSSION TITLE
Simultaneous RD Usage?

### DISCUSSION THREAD

**Discussion by Cap10Canuck** [itsToggle/plex_debrid#415] [General]
I am curious how compatible this solution would be if used simultaneously by multiple users off the same Plex Server.  In the past, I inadvertently accessed two files simultaneously from two different destinations and got a nastygram from RD telling me my account would be suspended if it happened again.  This was only because I had forgotten to deauthorize a device I had lent to a family member.  My current use of RD is now solely via RDT_Client as part of my *arr suite, but this use case here intrigues me!

**Comment by itsToggle** [comment]
As stated in the readme, you can share your plex server with an unlimted number of other users as all traffic is routed through your plex servers IP :)
